packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationSettings.tsx

```diff
@@ -74,9 +70,8 @@ const OrganizationDangerSection = () => {
 
   return (
     <ProfileSection
-      // title={localizationKeys('userProfile.start.profileSection.title')}
       // id='organization-danger'
-      title={'Danger'}
+      title={localizationKeys('organizationProfile.profilePage.dangerSection.title')}
```

Comment:
I think we haven't used the word "danger" yet - can you check if it makes sense to replace this one with `destructive`? You can check the `defaultEnglishResource` file for reference

Reply:
I've now realised that "danger" is the name of the section so you can ignore the above unless you think it makes more sense to go that way